lib/Sema/CSSimplify.cpp

```diff
@@ -950,8 +950,17 @@ ConstraintSystem::TypeMatchResult constraints::matchCallArguments(
   // Apply labels to arguments.
   SmallVector<AnyFunctionType::Param, 8> argsWithLabels;
   argsWithLabels.append(args.begin(), args.end());
+
+  // TODO: is this necessary?
   AnyFunctionType::relabelParams(argsWithLabels, argLabels);
```

Comment:
@slavapestov Do you remember why do we need to do this? Shouldn't labels in `args` always match the ones retrieved from expression by `getCalleeDeclAndArgs`?